resources/assets/js/components/ApplicationReview/ReviewApplications.tsx

```diff
@@ -80,7 +80,7 @@ const ReviewApplications: React.StatelessComponent<ReviewApplicationsProps> = ({
         </div>
       </div>
 
-      <div className="priority-alert">
+      {/* <div className="priority-alert">
```

Comment:
Can we remove anything we're no longer using? Assuming we don't plan on adding this back in anytime soon?

Reply:
Forgot to remove that.